package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "react.js_mentoring",
+  "version": "1.0.0",
+  "description": "ReactJS mentoring",
+  "main": "server.js",
+  "scripts": {
+    "test": "jest",
+    "sever start": "node server.js",
+    "start": "webpack --config webpack.config.js",
+    "build": "webpack"
```

Comment:
We will use production config too as I mentioned below. So you have to add additional `--config` for build too